### PR TITLE
Disable experiment

### PR DESCRIPTION
**Pull request by kamilszostak**
* Disable session storage buffer experiment
* Fix Data loss analyzer bug

### CODE REVIEW

File: JavaScript/JavaScriptSDK/DataLossAnalyzer.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@
         static reset() {
             if (DataLossAnalyzer.isEnabled()) {
                 Util.setSessionStorage(DataLossAnalyzer.ITEMS_QUEUED_KEY, "0");
+                Util.setSessionStorage(DataLossAnalyzer.ISSUES_REPORTED_KEY, "0");
```
Comment (MaxShehovtsov) [inline JavaScript/JavaScriptSDK/DataLossAnalyzer.ts]: Not sure why is this needed if you're disabling the analyzer

**Comment by MaxShehovtsov** [comment]
:shipit:

File: JavaScript/JavaScriptSDK/DataLossAnalyzer.ts (hunk 1 above)
Comment (kamilszostak) [inline JavaScript/JavaScriptSDK/DataLossAnalyzer.ts]: It will be useful if we want to enable the DataLossAnalyzer in the future.
